test/test_coordinates_space.py

```diff
@@ -0,0 +1,358 @@
+
+            return ht_list
+
+
+        # Checking if bbhx is installed, if not, then ignore this test
+        # and raise a warning.
+        # TODO: we need to implement a better way to install bbhx package.
+        bbhx_installed = is_module_installed('bbhx')
+        if not bbhx_installed:
+            passed = True
+            print("Ignore polarization test, because bbhx is not installed.")
+        else:
+            # cunstom E3
+            # All of those hard-coded numbers are carefully chosen,
+            # in order to almost co-align and co-locate both 2 detectors.
+            fine_tunning = 7992204.094271309
+            OMEGA_0 = 1.99098659277e-7
+            yangle = numpy.pi / 2 + fine_tunning * OMEGA_0
+            add_detector_on_earth(name='D1', longitude=1.8895427761465164,
+                                latitude=0.11450614784814996, yangle=yangle,
+                                xangle=yangle+numpy.pi/3, height=0)
+
+            # set parameters
+            params = {}
+            YRSID_SI = 31558149.763545603
+            params['ref_frame'] = 'SSB'
+            params['approximant'] = 'BBHX_PhenomD'
+            params['base_approximant'] = 'BBHX_PhenomD'
+            params['coa_phase'] = 0.0
+            params['mass1'] = 1e6
+            params['mass2'] = 8e5
+            params['spin1z'] = 0.0
+            params['spin2z'] = 0.0
+            params['distance'] = 410
+            params['inclination'] = numpy.pi/2  # edge-on
+            params['t_obs_start'] = 1*YRSID_SI
+            params['delta_f'] = 1./params['t_obs_start']
+            params['f_lower'] = 1e-4
+            params['f_ref'] = 8e-4
+            params['f_final'] = 0.1
+            params['delta_t'] = 1/0.2
+            params['t_offset'] = 9206958.120016199 # 0 degrees
+            t_lisa = YRSID_SI - params['t_offset'] + fine_tunning
+
+            nx = 100
+            longitude_array_high_res = numpy.linspace(
+                            0, 2*numpy.pi, num=nx, endpoint=False)
+
+            amp_E3_psi = []
+            phase_E3_psi = []
+            amp_LISA3_psi = []
+            phase_LISA3_psi = []
+
+            for polarization_lisa in numpy.linspace(
+                    0, 2*numpy.pi, endpoint=False, num=nx):
+                params['tc'], params['eclipticlongitude'], \
+                params['eclipticlatitude'], params['polarization'] = \
+                    lisa_to_ssb(t_lisa, 0, numpy.pi/4,
+                                polarization_lisa, params['t_offset'])
+
+                nparams = {'mass1':params['mass1'], 'mass2':params['mass2'],
+                            'spin1z':params['spin1z'],
+                            'spin2z':params['spin2z'],
+                            'f_lower':params['f_lower']}
+
+                bbhx_fd = get_fd_det_waveform(
+                            ifos=['LISA_A','LISA_E','LISA_T'], **params)
+                lisa_X_fd = -numpy.sqrt(2)/2 * bbhx_fd['LISA_A'] + \
```

Comment:
Thanks for the link; this `test_coordinates_space.py` is just testing for my transform, which overlaps LISA's X channel with ET's E1, when long wavelength approximation holds for the same signal, two V-shape detectors should have very similar detector response, so this can check if my frame transform is correct or not. It's not very matter here, just a test.